blog/2023-11-27-magazine-006.md

```diff
@@ -0,0 +1,195 @@
+![sourcemaps_and_stacktraces_after](./assets/2023-11-27-magazine-006/wing_sourcemaps_stacktraces_after.png)
+
+❤️ [Mark McCulloh](https://github.com/MarkMcCulloh) ❤️
+
+And there's even more excitement waiting for you! Feel free to explore the [The 23rd Wingly Update](https://www.twitch.tv/winglangio) for an extra dose of information. Join in and enjoy!
+
+### Upcoming Events
+
+**🔥 Live Workshop - January 3rd @4:30 - 6pm IST**
+Get hands-on with [Eyal Keren](https://github.com/ekeren) as he walks us through the process of constructing a full-fledged serverless workflow that ingests PDF files into one of Bedrock's supported foundation models, allowing you to query an AI agent and ask questions about the content of the file.
+For more information and to register, [click here](https://www.eventbrite.com/e/amazon-bedrock-winglang-tickets-769562721817?aff=oddtdtcreator).
+
+### The Wingly Update
+
+**[The Wingly Update](https://www.twitch.tv/winglangio)** is our biweekly stream where we share the latest developments of the project, chat with folks from the cloud industry, geek out and celebrate the beauty of the cloud.
+
+If you haven't been able to catch our show, you can find the complete stack of [all our episodes here](https://youtube.com/playlist?list=PL-P8v-FRassZBWsNoSafL_ReO0JO0xJVm&si=trffVrtGGMUZ-SKb). 
+
+Here are a few segments from the show we thought might be interesting to check out:
+
+- [The guest cornet with Ron Record](https://www.youtube.com/watch?v=i2ge8MNulqI): In this clip from The Wingly Update #23, [Elad](https://github.com/eladb) and [Eyal](https://github.com/ekeren) host [Ron](https://twitter.com/ronrecord) who is an open source developer for 30 years! In this video, Ron presents how he added two Wing workspaces registry with and without Neovim. These are Docker images designed for use with Kasm Workspaces, a container streaming platform that runs in your browser. 
+- [Winglang CHANGELOG Segment](https://www.youtube.com/watch?v=XA1jtNaJBrc): In this clip from The Wingly Update #23, [Uri](https://github.com/staycoolcall911) walks us through the latest updates in Winglang, such as expression as a map key, a React application within Wing concole, and more!
+- [Good first issue corner](https://www.youtube.com/watch?v=pS5BK18_7ys&t=1s): with [Revital](https://github.com/revitalbarletz)! In this clip from The Wingly Update #23, Revital and Eyal go through some of the Wing's `good first issues`, which provide an excellent starting point for contributors to become familiae with the Winglang framework.
+
+### Winglang Community Meeting
+
+**[The Winglang Community Meeting](https://meet.google.com/kcz-djtv-gvb?authuser=0&hs=122)** is our bi-weekly gathering where members of our community showcase cool apps, demos, and other projects! The upcoming session is scheduled for Tuesday, December 19th, at 2:30 PM UTC. We look forward to seeing you there!
```

Comment:
This link is to Google meet, can't we get a link to add the event to their calanders?

Reply:
These links are problematic, as if we change the meeting they dont get updated, perhaps better to add the link to our calendar ... If we would have a landing pages for these events it would be awesome!